docs/install.rst

```diff
@@ -197,54 +252,56 @@ GSISSH, but it requires (a) a MongoDB server and (b) a properly set-up
 passwordless SSH/GSISSH environment.
 
 
-MongoDB Server
---------------
+.. comments
+
+        MongoDB Server
+        --------------
 
-.. figure:: figures/hosts_and_ports.png
-     :width: 360pt
-     :align: center
-     :alt: MongoDB and SSH ports.
+        .. figure:: figures/hosts_and_ports.png
+             :width: 360pt
+             :align: center
+             :alt: MongoDB and SSH ports.
 
-The MongoDB server is used to store and retrieve operational data during the
-execution of an application using RADICAL-Pilot. The MongoDB server must
-be reachable on **port 27017** from **both**, the host that runs the
-Ensemble Toolkit application and the host that executes the MD tasks, i.e.,
-the HPC cluster (see blue arrows in the figure above). In our experience,
-a small VM instance (e.g., Amazon AWS) works exceptionally well for this.
+        The MongoDB server is used to store and retrieve operational data during the
+        execution of an application using RADICAL-Pilot. The MongoDB server must
+        be reachable on **port 27017** from **both**, the host that runs the
```

Comment:
**port 27017** -> on a specified port? I do not think we need to use 27017 right?